js/main.js

```diff
@@ -1,37 +1,77 @@
 'use strict'
 let canvas = document.getElementById("canvas");
-
 let ctx = canvas.getContext("2d");
+let score = 0;
+let foodImg = [];
+let randCount = Math.floor(Math.random()*5 + 1);
```

Comment:
```suggestion
let randCount = Math.floor(Math.random() * 5 + 1);
```
What is 5? Is it a magic number?